src/components/PopoverContent/PopoverContent.module.css

```diff
@@ -0,0 +1,138 @@
+/*------------------------------------*\
+    # POPOVER CONTENT
+\*------------------------------------*/
+
+/**
+ * A floating panel component with arbitrary content.
+ * Styling for the Popover component since the parent <Popover> component provides functionality.
+ */
+.popover-content {
+  z-index: var(--eds-z-index-top);
+}
+
+/**
+ * Main popover content, as opposed to the visual indicator arrow.
+ */
+.popover-content__main {
+  border-radius: var(--eds-border-radius-md);
+  overflow: auto;
+  box-shadow: var(--eds-box-shadow-md);
+  background-color: var(--eds-theme-color-background-neutral-default);
+}
+
+/**
+ * Decorative arrow pointing the Popover to the trigger element.
+ */
+.popover-content__arrow {
+  position: absolute;
+  width: 0;
+  height: 0;
+}
+
+/**
+ * Arrow directions, made into mixins for brevity in use.
+ */
+@define-mixin down-arrow {
+  border-top: var(--eds-size-half) solid
+    var(--eds-theme-color-background-neutral-default);
+  border-left: var(--eds-size-half) solid transparent;
+  border-right: var(--eds-size-half) solid transparent;
+}
+@define-mixin up-arrow {
+  border-bottom: var(--eds-size-half) solid
+    var(--eds-theme-color-background-neutral-default);
+  border-left: var(--eds-size-half) solid transparent;
+  border-right: var(--eds-size-half) solid transparent;
+}
+@define-mixin left-arrow {
+  border-right: var(--eds-size-half) solid
+    var(--eds-theme-color-background-neutral-default);
+  border-top: var(--eds-size-half) solid transparent;
+  border-bottom: var(--eds-size-half) solid transparent;
+}
+@define-mixin right-arrow {
+  border-left: var(--eds-size-half) solid
+    var(--eds-theme-color-background-neutral-default);
+  border-top: var(--eds-size-half) solid transparent;
+  border-bottom: var(--eds-size-half) solid transparent;
+}
+
+/**
+ * Arrow placement according to popover placement.
+ */
+:where([data-popper-placement='top-start'] > .popover-content__arrow) {
+  bottom: -0.25rem;
```

Comment:
Looks like `bottom: -0.25rem` is used wherever `@mixin down-arrow` is as well; could `bottom` be moved into `@mixin down-arrow`? Same for `top: -0.25rem` and `@mixin up-arrow`, `left: -0.25rem` and `@mixin left-arrow`, and `right: -0.25rem` and `@mixin right-arrow`.

Reply:
Oh snap good recognition, will move to the respective mixins